packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1120,3 +1132,68 @@ export async function pushDataToQueue(data: UploadType, requestQueueHandler: Req
 }
 
 export const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+
+export abstract class BStackLogger {
+    private static logFile = LOGS_FILE
+    private static logFilePath = path.join(process.cwd(), this.logFile)
+    private static logFolderPath = path.join(process.cwd(), 'logs')
```

Comment:
Can we get the `logs` constant from LOGS_FILE constant instead of maintaining two constants?